tests/test_slack_client.py

```diff
@@ -63,7 +63,14 @@ def test_check_command_format(self, slack_client):
         }
         format_check = slack_client.check_command_format(command)
         assert format_check.keys() >= {'valid', 'message', 'response_type'}
-        assert format_check['valid'] == True
+        assert format_check['valid'] is True
```

Comment:
nitpick here. Isn't `is true` redundant?
Might just need `assert format_check['valid']`